web_external/Viewer/ImageViewerWidgetWrapper/DivaGeoJSImageViewer.js

```diff
@@ -0,0 +1,111 @@
+
+import GeoJSImageViewerWidget from 'girder_plugins/large_image/views/imageViewerWidget/geojs';
+import ItemCollection from 'girder/collections/ItemCollection';
+
+var DivaGeoJSImageViewer = GeoJSImageViewerWidget.extend({
```

Comment:
can we rename it to VauiImageViewer. The only reason I am suggesting so that we are not tying our work to any specific project. I am fine with change (any other name) in a separate PR.